src/core/src/op/variadic_split.cpp

```diff
@@ -122,3 +119,28 @@ bool op::v1::VariadicSplit::has_evaluate() const {
     OV_OP_SCOPE(v1_VariadicSplit_has_evaluate);
     return get_input_element_type(1).is_integral_number() && get_input_element_type(2).is_integral_number();
 }
+
+bool op::v1::VariadicSplit::have_axis_and_splits_bound_set() const {
```

Comment:
Not sure about implementing this as a method of the VariadicSplit op, maybe it can be just kind of helper in the internal/anonymous namespace. What do you think?

Reply:
It is private helper. I don see benefit make it free function and pass pointer to operator to get its properties.